src/ring_theory/power_series/basic.lean

```diff
@@ -1132,6 +1157,26 @@ mv_power_series.mul_inv_of_unit φ u $ h
 
 end ring
 
+section comm_ring
+variables [comm_ring R]
+
+/-- The ring homomorphism taking a power series `f(X)` to `f(aX)`. -/
+noncomputable def eval_neg_hom : power_series R →+* power_series R :=
+eval_mul_hom (-1 : R)
```

Comment:
I think you should make `R` explicit on this one - there's no way for it to be inferred unlike `eval_mul_hom`.

Reply:
Done!

Reply:
>  I'm not so convinced that it's a good idea any more.

Thanks for counting them up - I think I agree, perhaps this wasn't worth it after all. Up to you and @laughinggas if you want to change it back to implicit